deployment/terraform/info.go

```diff
@@ -54,7 +54,14 @@ func displayInfo(output *Output) {
 	}
 
 	if output.HasProxy() {
-		fmt.Println("Proxy: " + output.Proxy.PublicIP)
+		if len(output.Proxies) > 1 {
+			fmt.Println("Proxies:")
+		} else {
+			fmt.Println("Proxy:")
+		}
+		for _, inst := range output.Proxies {
+			fmt.Println("- " + inst.PublicIP)
```

Comment:
Any reason for not using the name like we do for other instances?

Reply:
Thanks, fixed now.